packages/frontend/src/Home.jsx

```diff
@@ -47,139 +53,205 @@ export default observer(() => {
 
   if (!userContext.hasSignedUp) {
     return (
-      <div className="container">
-        <Button onClick={() => userContext.signup()}>Join</Button>
-      </div>
+      <div class='landing'>
```

Comment:
This component should probably be broken into it's own route. I think it's less of a problem if the SVGs aren't inlined.